File: 3515-find-if-digit-game-can-be-won/find-if-digit-game-can-be-won.java
class Solution {
    public boolean canAliceWin(int[] nums) {
        int singleDigitSum = 0;
        int doubleDigitSum = 0;
        for (int i = 0; i < nums.length; i++){
            String s = Integer.toString(nums[i]);
            if (s.length() % 2 == 1) singleDigitSum += nums[i];
            else doubleDigitSum += nums[i];
        }
        if (singleDigitSum > doubleDigitSum || doubleDigitSum > singleDigitSum) 
            return true;
        return false;
    }
}

File: 3515-find-if-digit-game-can-be-won/find-if-digit-game-can-be-won.py
class Solution:
    def canAliceWin(self, nums: List[int]) -> bool:
        singleDigitSum = 0
        doubleDigitSum = 0
        for i in nums:
            if len(str(i)) % 2 == 1:
                singleDigitSum += i
            else:
                doubleDigitSum += i
        if singleDigitSum > doubleDigitSum or doubleDigitSum > singleDigitSum:
            return True
        return False